checkClock/app/controllers/home_controller.rb

```diff
@@ -61,4 +61,11 @@ def out_employee
     end
   end
 
+  def getBranches 
+    branches = Branch.all
+    print(params[:valor])
+    return render json: {"message": branches }, status: :accepted
```

Comment:
I kind of imagine what you are attempting here 👀

Reply:
I tried to make an endpoint like a API rest